data_store/validation/pathfinders/cross_table_validation/consts.py

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime
+
+# Maps Pathfinder reporting periods from "Reporting period" and "Project finance changes" tables to start and end dates
+PFC_REPORTING_PERIOD_LABELS_TO_DATES = {
```

Comment:
What is PFC?

Reply:
Project finance changes - although maybe this is not ideally named, because the reporting period enums used in the "Project finance changes" table are also used in the "Reporting period" table. Feels better than just `REPORTING_PERIOD_LABELS_TO_DATES` though, which maybe is ambiguous given the reporting period headers in outputs, outcomes and forecast and actual spend tables? Not sure